templates/debian/default_isc-dhcp-server

```diff
@@ -8,4 +8,8 @@
 
 # On what interfaces should the DHCP server (dhcpd) serve DHCP requests?
 #       Separate multiple interfaces with spaces, e.g. "eth0 eth1".
+<% if @lsbmajdistrelease >= '9' && @lsbdistid == 'Debian' -%>
```

Comment:
Personally I prefer the facts `operatingsystem` and `operatingsystemmajrelease`. It looks like facterdb doesn't know the lsb versions either which causes the tests to fail.

Reply:
thank you . this is changed